install.sh

```diff
@@ -543,6 +543,17 @@ EOABORT
 )"
 fi
 
+# Multi-user support on macOS
+directories=("$HOMEBREW_REPOSITORY" "$HOMEBREW_CACHE")
+if [[ -z "${HOMEBREW_ON_LINUX-}" ]] && [[ -n "${HOMEBREW_ENABLE_MULTI_USER-}" ]]; then
```

Comment:
I think this should be done on Linux

Reply:
Linux doesn't unfortunately have a standard name for the `admin` group that I know of. Which group would we use? The primary group of the current user? Or we could create a `homebrew` group if one doesn't exist. That'd be the cleanest solution.

Reply:
`wheel` is the closest thing to a standard here I think.

Reply:
I wish it were:
* `wheel` seems to be used mainly on BSD-derived distros
* POSIX is silent on the matter
* LSB (Linux Standard Base) says `adm` is optional
* Debian and Ubuntu split the difference between `adm` and `staff`

I'm inclined to agree with @sjackman that creating a `homebrew` group is probably the best way forward on _any_ platform.

Reply:
I don't think `homebrew` grouping is a good way forward on macOS. You've convinced me it is on Linux, though.